packages/notifi-frontend-client/lib/NotifiFrontendClient.ts

```diff
@@ -1,12 +1,29 @@
 import { NotifiService } from '@notifi-network/notifi-graphql';
-import { UserFragmentFragment } from 'notifi-graphql/lib/gql/generated';
 
+import {
+  BeginLogInByTransactionResult,
+  CompleteLogInByTransactionInput,
+  CompleteLogInByTransactionMutation,
+  UserFragmentFragment,
+  WalletBlockchain,
+} from '../../notifi-graphql/lib/gql/generated';
```

Comment:
we should not use relative paths, since this is a different NPM package

Can you explore why your VS Code isn't resolving via the NPM workspaces?

Reply:
Ah, it does actually, just didn't realize that it was a best practice :)